eng/Testing.targets

```diff
@@ -1,3 +1,91 @@
 <Project>
 
+  <!--
+      Environment variables:
+        - IsGitHubActionsRunner:       indicates whether tests are currently run on GitHub Actions; computed, overridable
+        - IsAzdoRunner:                indicates whether tests are currently run on Azure DevOps; computed, overridable
+        - IsHelix:                     indicates whether tests are currently run on Helix; computed, overridable
+
+      Project capabilities:
+        - IsTestProject:               indicates whether the project is a test project; default is false; computed, overridable
+        - IncludeTestUtilities:        indicates whether the test project must not include the TestUtilities project reference; default is false; overridable
+        - RunsOnGithubActions:         indicates whether tests should run on GitHub Actions (either Windows or Linux); computed
+        - RunsOnGithubActionsWindows:  indicates whether tests should run on Windows in GitHub Actions; default is true; overridable
+        - RunsOnGithubActionsLinux:    indicates whether tests should run on Linux in GitHub Actions; default is true; overridable
+        - RunsOnAzdo:                  indicates whether tests should run on Azure DevOps; default is true; overridable
+        - RunsOnHelix:                 indicates whether tests should run on Helix (either Windows or Linux); computed
+        - RunsOnHelixWindows:          indicates whether tests should run on Windows in Helix; default is true; overridable
+        - RunsOnHelixLinux:            indicates whether tests should run on Linux in Helix; default is true; overridable
+    -->
+
+  <PropertyGroup>
+    <!-- See https://docs.github.com/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables -->
+    <IsGitHubActionsRunner Condition=" '$(IsGitHubActionsRunner)' == '' and '$(GITHUB_ACTIONS)' == 'true' ">true</IsGitHubActionsRunner>
+
+    <!-- See https://learn.microsoft.com/azure/devops/pipelines/build/variables#system-variables -->
+    <IsAzdoRunner Condition=" '$(IsAzdoRunner)' == '' and '$(SYSTEM_TEAMPROJECT)' != '' ">true</IsAzdoRunner>
+
+    <!-- See https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Helix/Sdk/Readme.md#common-helix-client-environment-variables -->
+    <IsHelix Condition=" '$(IsHelix)' == '' and '$(HELIX_CORRELATION_ID)' != '' ">true</IsHelix>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <RunsOnGithubActions>false</RunsOnGithubActions>
+    <RunsOnGithubActions Condition=" '$(RunsOnGithubActionsWindows)' == 'true' or '$(RunsOnGithubActionsLinux)' == 'true' ">true</RunsOnGithubActions>
+
+    <RunsOnHelix>false</RunsOnHelix>
+    <RunsOnHelix Condition=" '$(RunsOnHelixWindows)' == 'true' or '$(RunsOnHelixLinux)' == 'true' ">true</RunsOnHelix>
```

Comment:
Running on helix, and azdo are mutually exclusive. So, I think we should have property like `RunOnAzdo` with two possible values - `pipeline`, or `helix`. So, if we are running in Azdo, then this property would be relevant.

Reply:
Yes, this is handled in https://github.com/dotnet/aspire/pull/8518/files#diff-fb9d910fd40b64c36548859e4b6d19264d5b3bab9e79c515617181e760fb31b5 line:41